test/functional/test_framework/messages.py

```diff
@@ -1232,6 +1232,23 @@ def __repr__(self):
         return "msg_mempool()"
 
 
+class msg_notfound:
+    __slots__ = ("vec")
```

Comment:
This should be `__slots__ = ("vec",)` to force `tuple` instead of `str` :-)

Reply:
Ugh, I always get this wrong. Thx